jbpm-flow/src/main/java/org/jbpm/process/instance/ProcessImplementationPart.java

```diff
@@ -0,0 +1,7 @@
+package org.jbpm.process.instance;
```

Comment:
missing license header and would be nice to have line or two about this (what's the purpose and responsibility) in javadoc